modules/globebrowsing/tile/tile.h

```diff
@@ -109,7 +99,7 @@ class Tile : public cache::MemoryAwareCacheable {
     static const Tile TileUnavailable;
 
 private:
-    std::shared_ptr<ghoul::opengl::Texture> _texture;
+    ghoul::opengl::Texture* _texture;
```

Comment:
Would it makes sense to replace all of these with references to make sure that there is no `nullptr` anywhere?